packages/web-workers/src/create/worker.ts

```diff
@@ -1,14 +1,20 @@
-import {createEndpoint, Endpoint, MessageEndpoint} from '@remote-ui/rpc';
+import {
+  createEndpoint,
+  Endpoint,
+  MessageEndpoint,
+  CreateEndpointOptions,
+} from '@remote-ui/rpc';
 import {createWorkerMessenger} from '../messenger';
 import {createScriptUrl, FileOrModuleResolver} from './utilities';
 
-export interface CreateWorkerOptions {
+export interface CreateWorkerOptions<T> {
   createMessenger?(url: URL): MessageEndpoint;
+  endpointOptions?: CreateEndpointOptions<T>;
```

Comment:
Not sure if those should be spread into `CreateWorkerOptions ` instead ?

Reply:
Yeah, I'd rather `CreateWorkerOptions<T> extends CreateEndpointOptions<T>`